std_msgs/Header header
gelsight_ros/GelsightMarkers markers